plugins/flatpak/gs-flatpak.c

```diff
@@ -2139,6 +2190,18 @@ gs_flatpak_add_updates (GsFlatpak *self,
 			main_app = g_object_ref (app);
 		}
 
+		/* set the update highest update priority between the main app
+		 * and any important related apps (e.g. if we have a content
+		 * extension whose update should get prioritized, then we need to
+		 * set its main app priority to represent it) */
+		if (main_app != app) {
+			UpdatePriority main_app_update_priority =
+				get_app_update_priority (main_app);
+			if (update_priority < main_app_update_priority)
+				update_priority = main_app_update_priority;
+		}
+		set_app_update_priority (main_app, update_priority);
+
```

Comment:
I think we could drop this patch. Looking at the rationale in the commit message:

> For EOS, the priority for updates should be, from more important to less important:
>
> - Apps that have a "Content" runtime, which means they are likely news apps and need to be updated often;

We no longer update such apps regularly, particularly not news apps

> - OS updates;

OK I guess...

> - Apps from Endless' domain (since those are usually Knowledge Apps that also get updates with new content);

We no longer regularly update these apps.

> - Any other apps;
> - Runtimes.

Personally I would be happy with OS > app > runtime. But updating an app will, in practice, update its runtime(s), so…

Reply:
Yeah, I was feeling skeptical about this one as well, seeing as, indeed, the first half of its rationale is no longer a thing :) I'll drop it.